### ISSUE TITLE
Cannot instantiate interface Vertex\AddressValidationApi\Api\CleanseAddressInterface when using bin/magento queue:consumers:start async.operations.all

### ISSUE THREAD

**Issue by nbennett25** [magento/magento2#35743]
### Preconditions and environment
Following the documentation here:

https://devdocs.magento.com/guides/v2.4/config-guide/mq/aws-mq.html

- Magento 2.4.3-p1
- Vertex_AddressValidationApi module enabled
- Vertex_AddressValidation module disabled
- `amqp` configured with Amazon endpoint in app/etc/env.php, per the documentation linked above:
```
'queue' => [
        'consumers_wait_for_messages' => 0,
        'amqp' => [
            'host' => '[host]',
            'port' => '5671',
            'user' => '[user]',
            'password' => '[password]',
            'virtualhost' => '/',
            'ssl' => 'true'
        ]
    ],
```

### Steps to reproduce

Go to cli, run `bin/magento queue:consumers:start async.operations.all`

### Expected result

Queue runs as expected, no errors thrown.

### Actual result

Error is thrown:
`Cannot instantiate interface Vertex\AddressValidationApi\Api\CleanseAddressInterface#0 /[docroot]/vendor/magento/framework/ObjectManager/Factory/Compiled.php(108): Magento\Framework\ObjectManager\Factory\AbstractFactory->createObject()`

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [X] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by nbennett25** [comment]
@engcom-Hotel have you been able to reproduce the issue?

**Comment by engcom-Hotel** [comment]
Hello @nbennett25,

Still working on this. Will update you soon.

Thanks

**Comment by engcom-Hotel** [comment]
Hello @nbennett25,

Thanks for the report and collaboration!

But I would like to tell you that from releasing of Magento 2.4.4, the Vertex extension is no longer bundled with the core release. Please have a look at the below article:

https://docs.magento.com/user-guide/tax/taxes.html11

So I suggest you please connect with the Vertex developers directly.

Thanks

**Comment by engcom-Hotel** [comment]
Dear @nbennett25,

We have noticed that this issue has not been updated for a period of 14 Days. Hence we assume that this issue is fixed now, so we are closing it. Please raise a fresh ticket or reopen this ticket if you need more assistance on this.

Regards

**Comment by nbennett25** [comment]
@engcom-Hotel  - is there a patch available for 2.4.3 to resolve the error?

**Comment by engcom-Hotel** [comment]
Hello @nbennett25,

This code was removed from the code-base quite a while ago and we are not able to provide a fix or patch in the scope of this repository. Adobe Commerce and Magento Open Source are not supporting this module anymore for current and future versions.

As an advice, I would recommend trying:

- Contact Vertex extension support   and ask them if any patch or fix available

- If you are an Enterprise(Adobe Commerce) user, you should have an official Adobe Commerce Support service. Possibly make sense to contact them with this issue. I cannot guarantee but there is a chance that they helps

- Search for answer or similar issue in [Community Forums](https://community.magento.com/) or [Magento Stack Exchange](http://magento.stackexchange.com/)

Thanks
